setup.py

```diff
@@ -16,7 +16,7 @@
 
 class CompileProto(Command):
     # When adding new protobuf versions, also update: setup.cfg, python/arcticdb/__init__.py
-    _PROTOBUF_TO_GRPC_VERSION = {"3": "<=1.48.*", "4": ">=1.49"}
+    _PROTOBUF_TO_GRPC_VERSION = {"3": "<=1.30.*", "4": ">=1.49"}
```

Comment:
Can we add a Man Group detection just in case in someone else's environment there's only a version between 1.30 and 1.48?

Reply:
I think we can cross that bridge if we come to it, no? Anyway, it would only affect _contributors_ rather than users. Users downloading the wheel would still be fine.